app/components/post_list/post_list.js

```diff
@@ -27,14 +26,15 @@ export default class PostList extends PureComponent {
         }).isRequired,
         channelId: PropTypes.string,
         currentUserId: PropTypes.string,
+        highlightPostId: PropTypes.string,
         indicateNewMessages: PropTypes.bool,
         isSearchResult: PropTypes.bool,
-        lastViewedAt: PropTypes.number,
+        lastViewedAt: PropTypes.number, // Used by container // eslint-disable-line no-unused-prop-types
```

Comment:
as the last one if you don't need it then don't include it, the component will receive the prop anyways